lib/gui/models/flash-state.js

```diff
@@ -132,8 +133,8 @@ FlashState.service('FlashStateModel', function() {
         speed: _.attempt(() => {
           if (_.isNumber(state.speed) && !_.isNaN(state.speed)) {
 
-            // Transform bytes to megabytes preserving only two decimal places
-            return Math.floor(state.speed / 1e+6 * 100) / 100;
+            // Preserve only two decimal places
+            return Math.floor(units.bytesToMegabytes(state.speed) * 100) / 100;
```

Comment:
This code is really old, so I guess this was the "best" place to put it back then, but we can definitely rethink it. The precision-trimming is very nice to have in the DevTools output so the flash state output consist of a single line (otherwise DevTools hides the details and shows an arrow to show them, which makes seeing what is doing much harder), but we could also include some generic conversions on the DevTools analytics information output to trim long numbers.

I'm taking notes as well :)